pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -326,7 +303,18 @@ func (i *inode) Open(ctx context.Context, rp *vfs.ResolvingPath, vfsd *vfs.Dentr
 		return nil, syserror.EOVERFLOW
 	}
 
-	fd := fileDescription{}
+	var fd *fileDescription
```

Comment:
These code will conflict with FUSE_READ PR. This logic has already been implemented there. Which PR do we anticipate to merge first?